docs/semgrep-appsec-platform/jira.md

```diff
@@ -23,72 +23,137 @@ Object.entries(frontMatter).filter(
+Ensure a 1:1 mapping between the Jira issue type field values and the Semgrep values. 
 :::
 
-To create tickets for one or more Code findings, use the triage button from the Code findings page or from an individual finding's details page. Findings can be individually selected, or you can create tickets for all findings for a given rule. 
+### Sample mappings
+
+Semgrep's **Severity** field can have the following values: `Low`, `Medium`, `High`, `Critical`. The following Jira field setups would work for capturing this information:
+
+* A short text issue type field
+* A paragraph issue type field
+* A drop-down issue type field with the following options: `Low`, `Medium`, `High`, `Critical`
+* A checkbox issue type field with the following options: `Low`, `Medium`, `High`, `Critical`
+
+If you opt for a drop-down or a checkbox issue type field, verify that:
+
+* There are no misspellings
+* No valid options are missing. If your drop-down or checkbox issue type is missing the `Medium` option, Jira cannot create tickets for medium-severity findings.
+
+## Create tickets
+
+### Code
+
+To create tickets for one or more Code findings, you can use the triage button:
 
-Once a ticket has been created, a link will be present on the right side of the findings page and along the top of an individual finding's details page.
+* On the **Findings** page
+* On an individual finding's **Details** page.
+
+When working on the **Findings** page, you can select and create tickets for individual findings or all findings for a given rule.
+
+:::note
+Creating tickets for many findings at once may take some time. Tickets that take longer than 10 seconds to create are shown in the UI once you refresh the page.
+:::
 
 ![Create Jira ticket - Code](/img/jira-code-findings.png)
-**Figure** Code triage flow
+***Figure.*** Code triage flow
+
+Once a ticket has been created, a link appears on the right side of the **Findings** page and along the top of an individual finding's details page.
 
 ![Jira ticket created - Code](/img/jira-code-ticketed.png)
-**Figure** Code ticket created
+***Figure.*** Code ticket created
 
 ### Supply Chain
 
+The Supply Chain triage flow creates tickets for all usages of a given vulnerability. Usages can not be selected individually.
+
+To create tickets for one or more Supply Chain findings, you can use the triage button on the:
+
+* **Supply Chain > Vulnerabilities** page
+* Individual finding's **Details** page.
+
+![Create Jira ticket - Supply Chain](/img/jira-ssc-findings.png)
+***Figure*.** Supply Chain triage flow
+
+![Jira ticket created - Supply Chain](/img/jira-ssc-ticketed.png)
+***Figure*.** Supply Chain ticket created
+
 :::caution Limitations
 Tickets can only be created for Supply Chain findings with reachable usages. They cannot be created for **Always reachable** or **Unreachable** findings.
```

Comment:
...?? 
> Tickets can only be created for Supply Chain findings with reachable usages. They cannot be created for **Always reachable** or **Unreachable** findings.

@jsmoxon "can only be created for ... reachable usages" but not for "always reachable"?? Isn't always reachable our **most** reachable state?

Reply:
@zdhickman @jarrydlee can you weigh in here? I've heard conflicting things. My understanding is that once we move to the new SSC UI this will be moot but for now it's good to keep this callout explicit. Thanks!

Reply:
[Based on the discussion in Slack](https://semgrepinc.slack.com/archives/C02JT6YA5S7/p1714603223928329?thread_ts=1714511039.527759&cid=C02JT6YA5S7), the "reachable" is for all 3 types of reachable:
Always reachable, Conditionally reachable (reachable-if), "Code reachable" (which is not a formal term in the UI).

So the statement "They cannot be created for Always reachable" is false and should probably be updated.